.travis.yml

```diff
@@ -33,24 +35,17 @@ before_script:
   - sudo apt-get install google-chrome-beta
   - sudo apt-get install libstdc++6-4.7-dev
   - MIX_ENV=test mix compile --warnings-as-errors
+  - if [ -z ${WALLABY_DRIVER} ]; then travis_wait mix dialyzer --plt; fi # only run dialyzer for the env without drivers
```

Comment:
@keathley you seem to have removed this line in df1210215c7 - any particular reason or just an accident? :)